PersonApi.Tests/V1/Gateways/DynamoDbGatewayTests.cs

```diff
@@ -13,19 +13,20 @@
 
 namespace PersonApi.Tests.V1.Gateways
 {
-    [Collection("LogCall collection")]
+    [Collection("DynamoDb collection")]
     public class DynamoDbGatewayTests
     {
         private readonly Fixture _fixture = new Fixture();
-        private readonly Mock<IDynamoDBContext> _dynamoDb;
         private readonly Mock<ILogger<DynamoDbGateway>> _logger;
-        private readonly DynamoDbGateway _classUnderTest;
+        private DynamoDbGateway _classUnderTest;
 
-        public DynamoDbGatewayTests()
+        private readonly IDynamoDBContext _dynamoDb;
+
+        public DynamoDbGatewayTests(DynamoDbIntegrationTests<Startup> dbTestFixture)
```

Comment:
The 'integrationTests' file should be creating a table that can then be used when the SaveAsync() is called